lib/ccl/ccl-constants.js

```diff
@@ -1,5 +1,7 @@
-export const TC_PCR_VALIDITY_IN_HOURS = 72
-export const TC_RAT_VALIDITY_IN_HOURS = 48
+export const TC_PCR_VALIDITY_IN_HOURS = 24
+export const TC_RAT_VALIDITY_IN_HOURS = 24
```

Comment:
Shouldn't this change be back ported to the apps "Receive test results" feature (e.g. show RAT as out of date after 24h)?

Reply:
This is not yet decided.

Reply:
@mlenkeit 

Ok, thanks, but it is in the talks?

Reply:
Yes, but it will most likely stay as it is at the moment.